app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -120,6 +121,8 @@
 
     private static final int BUNDLE_MAX_SIZE = 300 * 1000; // 300K
 
+    private Handler uiHandler = new Handler(Looper.getMainLooper());
```

Comment:
Maybe we can just use ThreadUtils.postToMainThread() if this is a one-time thing?

Reply:
Ok, I didn't notice there's such a util class, I'll change to use that. Thx!